core/scripts/vrfv1/main.go

```diff
@@ -84,6 +88,56 @@ func main() {
 	//account.GasPrice = gp.Mul(gp, big.NewInt(2))
 
 	switch os.Args[1] {
+	case "coordinator-deploy":
+		cmd := flag.NewFlagSet("coordinator-deploy", flag.ExitOnError)
+		linkAddress := cmd.String("link-address", "", "LINK token contract address")
+		bhsAddress := cmd.String("bhs-address", "", "blockhash store contract address")
+		helpers.ParseArgs(cmd, os.Args[2:], "link-address", "bhs-address")
+		_, tx, _, err := solidity_vrf_coordinator_interface.DeployVRFCoordinator(
+			account, ec, common.HexToAddress(*linkAddress), common.HexToAddress(*bhsAddress))
+		helpers.PanicErr(err)
+		fmt.Println("Coordinator deployment tx:", helpers.ExplorerLink(chainID, tx.Hash()))
+		coordinatorAddress, err := bind.WaitDeployed(context.Background(), ec, tx)
+		helpers.PanicErr(err)
+		fmt.Println("Coordinator address:", coordinatorAddress)
```

Comment:
Note: `ConfirmContractDeployed` can replace this logic but it's still in the testnet directory on develop.

Reply:
Yeah, that stuff is refactored out into the `common` package in one of Jin's PR's. I wonder if I should just refactor here and pull that stuff in, it might end up with less merge conflicts that way?